### ISSUE TITLE
Optimize build

### ISSUE THREAD

**Issue by 0mkara** [quantanet/ethcode#18]
Possibly we can optimize the build and get rid of this warning.
```
This extension consists of 29499 separate files. For performance reasons, you should bundle your extension: https://aka.ms/vscode-bundle-extension. You should also exclude unnecessary files by adding them to your .vscodeignore: https://aka.ms/vscode-vscodeignore
```

**Comment by Ayanrocks** [comment]
Where to get this error?

**Comment by 0mkara** [comment]
`vsce package --yarn`

**Comment by Ayanrocks** [comment]
Fixed in #21
